README.md

```diff
@@ -45,11 +45,28 @@ This stateless service demostrates a watch dog application which is monitoring t
 For compiling the samples use gradle. This should be configured as part of the Service Fabric Java SDK installation.
 
 ## Deploying the samples
-All the samples once compiled can be deployed immediately using the install.sh scripts located inside Scripts folder of every sample. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 
+All the samples once compiled can be deployed immediately using the install.sh scripts located inside Scripts folder of every sample. You need to trigger them from the project root-level. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 
 
 ## Compiling and Deploying the samples using Service Fabric Plugin for Eclipse
-The Service Fabric Plugin for Eclipse can be used to compile and deploy the samples, from Eclipse. To know more about this you can refer to our documentation - 
-[Import and Deploy github java samples using Service Fabric Eclipse plugin][service-fabric-github-samples-with-eclipse].
+
+### Import and Deploy GitHub Java Samples on Linux using Service Fabric Eclipse plugin
+1. Clone the repository in your dev-box - ``https://github.com/Azure-Samples/service-fabric-java-getting-started.git``
+2. On your Eclipse, you need to make sure, Service Fabric Plugin installed.
+3. Create a new workspace.
+4. Open the ``Service Fabric`` Perspective. You can change your Eclipse perspective by following the path - ``Window => Perspective => Open Perspective => Other`` and select  ``Service Fabric`` from the list you see.
+5. To import the GitHub samples in Eclipse, you can go to ``File => Import => General => Existing Project into Workspace`` and select the path to say ``EchoServer1.0``, from your cloned GitHub samples.
+6. Now to build and deploy you can follow the steps mentioned above in the document [Build and deploy the Service Fabric application using Eclipse](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-eclipse#build-and-deploy-the-service-fabric-application-using-eclipse).
+
+### Import and Deploy Github Java samples on Mac using Service Fabric Eclipse plugin
+The steps mentioned above for Linux, mostly hold true here as well, with minor modifications as mentioned below.

```

Comment:
the steps mentioned for linux.. just above..  just above the mac section, there is the linux section.. 
so referring to that should be fine. I guess we don't need to change anything here